### ISSUE TITLE
Error deleting setting in XML parser

### ISSUE THREAD

**Issue by RazuvaevDD** [RazuvaevDD/ProjectWork2019Neiro#27]
**Entry conditions**
_Settings.xml_
```
<root>
    <setting id="2">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
</root>
```
_main.cpp_
```
Settings_module::XMLParser p("Settings.xml");
p.loadXML();
p.deleteSetting(1);
```
**expected result**
Settings.xml has not changed;
Function returned false.

**current result**
_Settings.xml_
`<root/>`
Function returned true.

**Comment by AverinLV** [comment]
Fixed in 3b7cb5c

**Comment by RazuvaevDD** [comment]
Ok
